### PR TITLE
style: Add Jest linting to ESLint

### PR DESCRIPTION
**Pull request by rafenden**
Add linting for Jest tests using https://github.com/jest-community/eslint-plugin-jest

### CODE REVIEW

**Comment by rafenden** [comment]
> This looks like it's doing more than just adding lint rules, should this be two 

This PR also has to apply the new rules to pass the tests.

File: src/activity-feed/__tests__/ActivityFeedPagination.test.jsx [hunk 1]
```diff
@@ -17,13 +17,6 @@ describe('ActivityFeedPagination', () => {
     expect(tree).toMatchSnapshot()
   })
 
-  test('renders pagination with loader', () => {
```
Comment (rafenden) [inline src/activity-feed/__tests__/ActivityFeedPagination.test.jsx]: Duplicated test.

**Comment by debitan** [comment]
It looks like some tests are deleted entirely, is this expected?

**Comment by rafenden** [comment]
> It looks like some tests are deleted entirely, is this expected?

These tests were duplicated and the linter detected it.

**Pull request merged**